src/components/shared/Extensions/Toast/partials/CloseButton.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { CloseButtonProps } from 'react-toastify';
+import Icon from '~shared/Icon';
+
+const CloseButton = ({ closeToast }: CloseButtonProps) => (
```

Comment:
Shouldn't be
`const CloseButton: FC<CloseButtonProps> = ({ closeToast }) => ...` ?